.travis.yml

```diff
@@ -49,9 +50,17 @@ deploy:
     on:
       branch: master
       condition: $TRAVIS_R_VERSION_STRING == "release"
-  # our image to dockerhub
+  # our image to dockerhub for tags
   - provider: script
-    script: bash docker_push
+    script: bash docker_push qmongr
+    skip_cleanup: true
+    on:
+      tags: true
+      branch: master
+      condition: $TRAVIS_R_VERSION_STRING == "release"
```

Comment:
This will probably not work as intended: https://docs.travis-ci.com/user/deployment-v2/conditional#tag

> Setting `tags` to `true` causes the `branch` condition to be ignored, otherwise `$TRAVIS_TAG` is ignored, and the `branch` condition is considered.

I wanted to deploy tags, but only on master.